index.html

```diff
@@ -38,37 +38,76 @@ <h3 class="description"><a href="https://servo.org/"><b>Servo</b></a>
 
-                <a class="btn btn-primary btn-large" role="button"
+                <a class="btn btn-primary btn-large" id="button-linux" role="button"
                    href="nightly/linux/servo-latest.tar.gz">Linux Build (64-bit)</a><br>
             </span>
-            <span class="col-sm-push">
-                <a class="btn" href="nightly/windows-msvc/servo-latest.exe.sha256">sha256</a><br>
-
-                <a class="btn" href="nightly/windows-msvc/servo-latest.zip.sha256">sha256</a><br>
-
-                <a class="btn" href="nightly/uwp/servo-latest.zip.sha256">sha256</a><br>
-
-                <a class="btn" href="nightly/mac/servo-latest.dmg.sha256">sha256</a><br>
-
-                <a class="btn" href="nightly/linux/servo-latest.tar.gz.sha256">sha256</a><br>
-            </span>
+        </div>
+    </div>
+    <div class="row" id="build-info-row">
+        <div class="col-sm-6 col-sm-offset-3">
+            <table class="table" id="build-info">
+                <thead>
+                    <tr>
+                        <th scope="col">Platform</th>
+                        <th scope="col">Build date</th>
+                        <th scope="col">SHA-256</th>
+                    </tr>
+                </thead>
+                <tbody>
+                    <tr>
+                        <th scope="row">Windows Installer</th>
+                        <td id="date-windows-exe">?</td>
+                        <td>
+                            <a id="sha256-windows-exe" href="nightly/windows-msvc/servo-latest.exe.sha256">sha256</a>
+                        </td>
+                    </tr>
+                    <tr>
+                        <th scope="row">Windows Zip</th>
+                        <td id="date-windows-zip">?</td>
+                        <td>
+                            <a id="sha256-windows-zip" href="nightly/windows-msvc/servo-latest.zip.sha256">sha256</a>
+                        </td>
+                    </tr>
+                    <tr>
+                        <th scope="row">UWP</th>
+                        <td id="date-uwp">?</td>
+                        <td>
+                            <a id="sha256-uwp" href="nightly/uwp/servo-latest.zip.sha256">sha256</a>
+                        </td>
+                    </tr>
+                    <tr>
+                        <th scope="row">macOS</th>
+                        <td id="date-macos">?</td>
+                        <td>
+                            <a id="sha256-macos" href="nightly/mac/servo-latest.dmg.sha256">sha256</a>
+                        </td>
+                    </tr>
+                    <tr>
+                        <th scope="row">Linux</th>
+                        <td id="date-linux">?</td>
+                        <td>
+                            <a id="sha256-linux" href="nightly/linux/servo-latest.tar.gz.sha256">sha256</a>
+                        </td>
+                    </tr>
+                </tbody>
+            </table>
```

Comment:
One issue is that the build dates column's position flickers when the dates load. Is there something wrong with my Bootstrap code?

Reply:
I also don't love that the row lines extend unnecessarily to the right. I'd prefer that they aren't there at all, or that they're at least equal on the two sides. I'm relatively inexperienced with Bootstrap, so I'm not sure how to fix it.

Reply:
I think the hashes should probably in a code block, but when I tried that, the link styling looked weird. Maybe I should create a custom class for a plain code block (like the GitHub ones)?

Reply:
I don't know anything about using bootstrap, so I won't be able to meaningfully assist with the styling issues.